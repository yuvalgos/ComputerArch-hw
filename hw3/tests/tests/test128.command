./dflow_calc tests/test128.dat tests/test128.in d65 d13 d191 p167 d56 d188 d43 p53 d44 d160 p52 p48 p177 p131 p86 p195 d118 p102 d194 p0 d71 d180 d185 p123 d28 d51 p147 d137 d54 p12 d136 d27 d39 p178 d115 p129 d7 d111 d157 d127 p130 d158 d103 d147 p44 d115