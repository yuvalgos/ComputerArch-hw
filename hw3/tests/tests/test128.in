5 16 22 5
2 24 27 5
2 11 18 6
0 27 8 25
2 26 17 13
6 11 22 19
3 9 20 22
7 20 19 8
0 23 31 28
3 4 22 21
2 7 13 19
4 11 7 2
2 24 5 3
2 18 22 18
8 12 5 4
7 9 15 14
2 29 20 7
4 4 31 25
8 17 6 20
5 21 30 22
7 14 10 19
6 11 15 3
4 31 20 29
4 26 14 10
5 29 30 2
5 30 13 25
3 28 24 29
3 10 12 20
5 10 2 10
7 27 7 18
2 25 28 17
8 4 9 26
0 23 3 26
7 1 0 17
5 5 5 20
4 26 19 16
6 1 14 23
5 21 13 0
2 13 31 24
6 19 9 17
5 0 26 31
2 12 24 19
7 6 27 27
7 3 20 14
3 7 17 23
5 30 9 18
0 14 24 19
6 9 13 9
5 26 29 1
6 21 0 23
5 8 11 16
8 6 16 25
3 0 12 17
1 18 28 12
0 20 25 13
6 13 26 9
8 19 4 27
1 18 28 25
3 8 6 0
5 12 17 8
4 15 8 17
3 16 30 5
3 28 12 23
1 9 30 29
7 9 31 13
4 27 17 1
1 14 17 9
8 0 26 20
1 0 7 12
7 11 22 12
1 9 28 29
0 31 11 22
3 23 31 23
6 22 22 9
8 25 16 20
2 15 13 8
4 30 0 27
5 4 27 8
4 29 11 4
2 12 12 26
0 25 24 3
6 30 5 14
1 10 3 18
3 10 18 24
1 14 19 15
6 1 21 18
4 26 26 1
4 1 1 10
7 12 29 9
1 8 9 6
8 31 7 4
4 23 21 30
1 2 14 18
6 23 24 3
5 23 11 29
2 10 23 24
4 30 0 23
5 21 31 5
7 23 31 5
0 13 23 9
6 5 28 8
4 20 26 11
2 27 4 30
3 1 12 10
7 3 1 10
0 9 7 11
0 14 28 10
8 0 29 26
7 3 24 5
6 8 25 10
8 8 12 14
4 12 8 15
6 24 29 31
0 30 28 19
4 11 12 17
7 31 6 1
2 11 23 31
7 22 4 31
3 0 22 7
0 3 30 12
4 18 15 1
1 31 20 29
4 12 9 15
7 20 3 3
7 5 30 3
1 23 1 7
5 24 19 7
7 12 16 19
4 23 14 15
8 8 31 22
2 18 3 4
3 13 14 22
0 0 10 30
0 11 22 11
6 9 19 5
8 5 17 5
3 11 18 22
5 2 4 7
4 24 9 14
4 14 9 9
5 4 27 23
0 10 20 8
7 7 3 31
0 24 1 10
5 10 3 10
0 16 1 14
3 7 28 24
5 29 29 14
5 8 23 1
5 18 1 6
0 30 5 14
1 22 20 18
5 19 14 9
2 6 28 20
0 7 16 11
2 0 1 12
7 28 29 17
6 0 29 2
7 15 29 11
6 11 15 10
3 2 12 30
1 5 6 17
1 2 28 9
8 21 8 4
1 18 10 0
5 4 16 28
3 25 12 29
3 10 8 20
6 16 17 31
8 9 20 16
6 6 29 16
7 3 16 27
7 29 8 6
7 16 16 0
8 17 28 25
1 29 18 13
1 3 31 11
4 15 19 10
3 16 16 9
4 5 11 22
7 12 20 12
1 20 22 13
5 18 17 27
1 0 4 15
6 21 30 10
8 18 20 29
5 25 19 27
0 3 24 27
6 24 11 10
5 6 28 0
5 12 16 9
3 16 4 18
2 8 3 23
2 22 17 3
3 19 10 12
1 8 4 20
2 11 28 27
4 17 12 26
0 5 1 11
4 21 18 12
