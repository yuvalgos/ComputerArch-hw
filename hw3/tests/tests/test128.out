Reading the opcodes latency info from tests/test128.dat ... Got latency for 9 opcodes
Reading the program file tests/test128.in ... Found 200 instructions
getProgDepth()==304
getInstDeps(65)=={18,36}
getInstDeps(13)=={-1,-1}
getInstDeps(191)=={165,185}
getDepDepth(167)==196
getInstDeps(56)=={31,29}
getInstDeps(188)=={159,167}
getInstDeps(43)=={7,20}
getDepDepth(53)==64
getInstDeps(44)=={18,32}
getInstDeps(160)=={127,150}
getDepDepth(52)==64
getDepDepth(48)==49
getDepDepth(177)==237
getDepDepth(131)==185
getDepDepth(86)==82
getDepDepth(195)==269
getInstDeps(118)=={117,44}
getDepDepth(102)==145
getInstDeps(194)=={167,190}
getDepDepth(0)==0
getInstDeps(71)=={69,-1}
getInstDeps(180)=={123,127}
getInstDeps(185)=={181,175}
getDepDepth(123)==177
getInstDeps(28)=={-1,27}
getInstDeps(51)=={0,30}
getDepDepth(147)==224
getInstDeps(137)=={77,44}
getInstDeps(54)=={30,38}
getDepDepth(12)==0
getInstDeps(136)=={130,117}
getInstDeps(27)=={14,7}
getInstDeps(39)=={15,18}
getDepDepth(178)==273
getInstDeps(115)=={51,103}
getDepDepth(129)==185
getInstDeps(7)=={-1,-1}
getInstDeps(111)=={110,75}
getInstDeps(157)=={147,137}
getInstDeps(127)=={61,56}
getDepDepth(130)==177
getInstDeps(158)=={147,136}
getInstDeps(103)=={88,95}
getInstDeps(147)=={78,139}
getDepDepth(44)==42
getInstDeps(115)=={51,103}
